src/lib/init.ts

```diff
@@ -561,15 +561,18 @@ function stringifyError(e${isJS ? "" : ": unknown"}) {
 }
 
 function getGatewayIndexJsContent(isJS: boolean) {
-  return `import { Fluence } from "@fluencelabs/js-client";
+  return `import { Fluence, KeyPair } from "@fluencelabs/js-client";
 import relays from "../relays.json" assert { type: "json" };
 import { ${isJS ? "" : "type Static, "}Type } from "@sinclair/typebox";
 ${isJS ? "" : 'import { type FastifyInstance } from "fastify";'}
 
 import { helloWorld, helloWorldRemote, showSubnet, runDeployedServices } from "../compiled-aqua/main.js";
 
 const DEFAULT_ACCESS_TOKEN = "abcdefhi";
-const DEFAULT_PEER_PRIVATE_KEY = new Array(32).fill("a").join("");
+
+const DEFAULT_PEER_PRIVATE_KEY = Buffer.from(
+  (await KeyPair.randomEd25519()).toEd25519PrivateKey(),
+).toString("base64");
```

Comment:
Do you really need this? is this for an example or something. It doesn't look so good. Maybe js-client api needs to be a bit more user friendly in this regard. Anyway, since you are using a random key, probably you can remove all that, I was under impression js-client uses random key by default so this argument can be ommited

Reply:
I do this to be more explicit, e.g., to show current private peer in server logs.

This snippet doesn't look good, yes. Because i need to convert private key accepted by JS client to CLI format and backwards. User generates private key via CLI and copies it to env variable. Probably from that perspective it's kinda ok.